### PR TITLE
faac: conan v2 support

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

---

- [ ] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/faac/all/conandata.yml [hunk 1]
```diff
@@ -8,4 +8,5 @@ sources:
 patches:
   "1.30":
     - patch_file: "patches/001-fix-out-of-root-build.patch"
-      base_path: "source_subfolder"
+      patch_description: "Fix out of root build"
```
Comment (prince-chrismc) [inline recipes/faac/all/conandata.yml:11]: https://github.com/conan-io/conan-center-index/blob/39b8177ad0ecbec6aa91fbaf174c1ecec8de0859/recipes/faac/all/patches/001-fix-out-of-root-build.patch#LL6C6-L6C101

```suggestion
      patch_description: "Fix out of root build"
      patch_source: "https://github.com/m-ab-s/media-autobuild_suite/commit/11d58abd8fde7c70378993029bad7bbaeb5f7e3e"
```

**Review by prince-chrismc — COMMENTED** [review]
Just the patch source pretty please! 🙏 ❤️

File: recipes/faac/all/conandata.yml (hunk 1 above)
Comment (SpaceIm) [reply recipes/faac/all/conandata.yml:11]: And when you look at this link, it seems totally unrelated. This is why I've skipped this field.

Comment (prince-chrismc) [reply recipes/faac/all/conandata.yml:11]: 🙈 Quality Control ... why did that go in the first place lol -- Okay well I answered my own question :)

**Review by prince-chrismc — DISMISSED** [review]

File: recipes/faac/all/conandata.yml (hunk 1 above)
Comment (SpaceIm) [reply recipes/faac/all/conandata.yml:11]: this patch maybe https://github.com/knik0/faac/commit/c8d12a5c7c5b6f1c4593f0a6c1eeceacc4d7c941.patch?

Comment (prince-chrismc) [reply recipes/faac/all/conandata.yml:11]: 😈 

```suggestion
      patch_description: "Fix out of root build"
      patch_source: "https://github.com/knik0/faac/commit/c8d12a5c7c5b6f1c4593f0a6c1eeceacc4d7c941.patch"
```

Now we can add it 🚀

**Review by prince-chrismc — APPROVED** [review]

**Review by danimtb — APPROVED** [review]

**Pull request merged**